ports/windows/Makefile

```diff
@@ -1,11 +1,30 @@
+# Select the variant to build for.
+VARIANT ?= standard
+
+# If the build directory is not given, make it reflect the variant name.
+BUILD ?= build-$(VARIANT)
+
+VARIANT_DIR ?= variants/$(VARIANT)
+ifeq ($(wildcard $(VARIANT_DIR)/.),)
+$(error Invalid VARIANT specified: $(VARIANT_DIR))
+endif
+
 include ../../py/mkenv.mk
 -include mpconfigport.mk
+include $(VARIANT_DIR)/mpconfigvariant.mk
+
+# use FROZEN_MANIFEST for new projects, others are legacy
+FROZEN_MANIFEST ?= variants/manifest.py
+FROZEN_DIR =
+FROZEN_MPY_DIR =
 
-# define main target
-PROG = micropython
+# Define main target
+# This should be configured by the mpconfigvariant.mk
```

Comment:
This was originally just `micropython` (as it is in this file) but I just noticed in the mpconfigvariant.mk files add the `.exe` extension. There's a number of issues related to that already so for now it's better to have the variants do the same and use the name without extension.